sdk/monitor/monitor-query/review/monitor-query.api.md

```diff
@@ -0,0 +1,293 @@
+    definitions: MetricDefinition[];
+}
+
+// @public
+export interface GetMetricNamespacesOptions {
+    startTime?: string;
+}
+
+// @public
+export interface GetMetricNamespacesResponse {
+    namespaces: MetricNamespace[];
+}
+
+// @public
+export interface LocalizableString {
+    localizedValue?: string;
+    value: string;
+}
+
+// @public
+export class LogsClient {
+    constructor(tokenCredential: TokenCredential, options?: LogsClientOptions);
+    // (undocumented)
+    queryLogs(workspaceId: string, query: string, options?: QueryLogsOptions): Promise<QueryLogsResult>;
+    // (undocumented)
+    queryLogsBatch(batch: QueryLogsBatch, options?: QueryLogsBatchOptions): Promise<QueryLogsBatchResponse>;
+}
+
+// @public (undocumented)
+export interface LogsClientOptions extends PipelineOptions {
+    host?: string;
+}
+
+// @public
+export interface MetadataValue {
+    name?: LocalizableString;
+    value?: string;
+}
+
+// @public
+export interface Metric {
+    displayDescription: string;
+    errorCode?: string;
+    id: string;
+    name: LocalizableString;
+    timeseries: TimeSeriesElement[];
+    type: string;
+    unit: MetricUnit;
+}
+
+// @public
+export interface MetricAvailability {
+    retention?: string;
+    timeGrain?: string;
+}
+
+// @public
+export interface MetricColumn {
+    name?: string;
+    type?: ColumnDataType;
+}
+
+// @public
+export interface MetricDefinition {
+    category?: string;
+    dimensions?: LocalizableString_2[];
+    displayDescription?: string;
+    id?: string;
+    isDimensionRequired?: boolean;
+    metricAvailabilities?: MetricAvailability[];
+    // Warning: (ae-forgotten-export) The symbol "LocalizableString" needs to be exported by the entry point index.d.ts
+    name?: LocalizableString_2;
+    primaryAggregationType?: AggregationType;
+    resourceId?: string;
+    // Warning: (ae-forgotten-export) The symbol "MetricUnit" needs to be exported by the entry point index.d.ts
+    unit?: MetricUnit_2;
+}
+
+// @public
+export interface MetricNamespace {
+    classification?: NamespaceClassification;
+    id?: string;
+    name?: string;
+    properties?: MetricNamespaceName;
+    type?: string;
+}
+
+// @public
+export interface MetricNamespaceName {
+    metricNamespaceName?: string;
+}
+
+// @public (undocumented)
+export class MetricsClient {
+    constructor(tokenCredential: TokenCredential, options?: MetricsClientOptions);
+    // (undocumented)
+    getMetricDefinitions(resourceUri: string, options?: GetMetricDefinitionsOptions): Promise<GetMetricDefinitionsResponse>;
+    // (undocumented)
+    getMetricNamespaces(resourceUri: string, options?: GetMetricNamespacesOptions): Promise<GetMetricNamespacesResponse>;
+    // (undocumented)
+    queryMetrics(resourceUri: string, options?: QueryMetricsOptions): Promise<QueryMetricsResponse>;
+}
+
+// @public (undocumented)
+export interface MetricsClientOptions extends PipelineOptions {
+    $host?: string;
```

Comment:
can we rename this? `$host` is a little finnicky to declare in JS literals

Reply:
Yeah, was thinking 'endpoint' would be more appropriate based on other clients.

Reply:
Renamed to 'endpoint'